### DISCUSSION TITLE
Can I use "kiennq" `shim.exe` with shovel? (To avoid ^C issue of powershell in Windows Terminal)

### DISCUSSION THREAD

**Discussion by david-bakin** [Ash258/scoop#202] [Q&A] [answered]
I see https://github.com/lukesampson/scoop/pull/3998 (add alt shim support) is in the TODO list of https://github.com/Ash258/Scoop-Core/issues/27 (backports tracker) - does that mean that the ^C problem with powershell 7 (especially inside of Windows Terminal) is not yet fixed?

If not, if I was to simply drop `....\scoop\apps\scoop\current\supporting\shims\kiennq\shim.exe` from a scoop installation on top of `...\scoop\apps\scoop\current\supporting\shimexe\bin\shim.exe` of a shovel upgrade, would that just work?  (I myself am only interested in Windows on x64.)

**Comment by Ash258** [comment, marked as answer]
Any of the random repos containing "alternative" shims, which were built manually without providing any proof of built or anything, will not be used in any of Shovel related repositories.

Letting user to have option to use any of it is bad practice and does not make sense.

If you want it, do whatever you need. You will not receive any support from me.
